### DISCUSSION TITLE
actions proxy error with vue2.7 in IE11

### DISCUSSION THREAD

**Discussion by xm-18y** [vuejs/pinia#1440] [Help and Questions] [answered]
### Reproduction

https://github.com/FE-Roading/vue2.7-element-admin

### Steps to reproduce the bug

use Vue27 and setup mode
```html
<script lang="ts" setup>
````
related code was below:
```vue
  const { login } = useAppStore()
  function onLogin() {
    loginFormRef?.value?.validate((valid) => {
      if (valid) {
        loading.value = true
        const params = unref(loginForm)
        debugger
        login(params)  // ---------------Error at this step
          .then(() => {
            loading.value = false
            router.push("/")
          })
          .catch(() => {
            loading.value = false
          })
      } else {
        return false
      }
    })
  }
```
![Error Capture](https://s3.bmp.ovh/imgs/2022/07/10/029b4d667c9f7cc9.png)

### Expected behavior

Same code worked in Chrome111. 

### Actual behavior

capture error in ie11.

### Additional information

Windows Version is Window10H2, IE version is `11.789.19041.0`.

**Comment by dangc9** [comment]
not supported ？

**Comment by posva** [comment, marked as answer]
I added ca73db9 to remove the devtools plugin in environments without Proxy.
IE is supported but devtools and other DX features can only be enabled in modern browsers

**Comment by weijuer** [comment]
How to resolve it with webpack
